tools/launchdev.sh

```diff
@@ -59,12 +61,10 @@ function init(){
         ST2_REPO=${CURRENT_DIR}/${COMMAND_PATH}/..
     fi
 
-    VIRTUALENV=${VIRTUALENV:-${ST2_REPO}/virtualenv}
```

Comment:
I think this merge conflict resolution didn't work out since this file is missing some changes from master.